articles/appliance/admin/inviting-coadmins.md

```diff
@@ -0,0 +1,19 @@
+# Appliance Administration: Inviting Co-Administrators
+
+You may invite additional users to become co-administrators of your Appliance instances. This is done via the Appliance configuration area of the Management Dashboard.
+
+> The User whom you'd like to be added as a co-administrator must be have an existing entity in the Appliance.
+
+![](https://auth0-1.wistia.com/medias/2t8n98qc5j)
+
+1. Switch over to the App Tenant so that you will have access to the Appliance configuration options.
```

Comment:
We need a step here where you explain that the administrator needs to be created in the RTA first.